src/index.js

```diff
@@ -1580,10 +1583,11 @@ module.exports = {
 			this.settings.routes.forEach(route => this.addRoute(route));
 
 		// Regenerate all auto aliases routes
+		const debounceTime = this.settings.debounceTime >= 0 ? parseInt(this.settings.debounceTime) : 500;
```

Comment:
`this.settings.debounceTime >= 0` is not correct. Just `this.settings.debounceTime > 0`